src/main/resources/META-INF/create.sql

```diff
@@ -8,7 +8,19 @@ create table if not exists SecurityGroup ( id bigint not null, name varchar(255)
 create table if not exists SecurityGroupInterface ( id bigint not null, name varchar(255), policyId varchar(255), tag varchar(255));
  
 create table if not exists VSSDevice ( id bigint not null, name varchar(255), primary key (id) );
+
+create table if not exists DomainEntity ( Id bigint not null, name varchar(255), appliance_manager_connector_fk bigint, primary key (Id));
```

Comment:
i would just name the table domain/policy etc and remove the domain suffix. this also makes it consistent with the other tables